### PR TITLE
Update blight code

### PR DESCRIPTION
**Pull request by WoodardEric**
please pull into master

### CODE REVIEW

File: LevelMain.cpp [hunk 1]
```diff
@@ -134,15 +134,17 @@ Export void AIProc()
...
+	LOCATION blightSpawnLoc(1 + X_, 255 + Y_);
+	
+
+	GameMap::SetVirusUL(blightSpawnLoc, 1); // Spawn the blight
+	TethysGame::SetMicrobeSpreadSpeed(60);	// set blight speed
 
-	// Warning Message
-	TethysGame::AddMessage(1248, 576, "Microbe growth detected!", -1, 152);
+	TethysGame::AddMessage(1 + X_, 255 + Y_, "The Blight is Approaching", -1, 152); // Check Sound Message
+
```
Comment (Brett208) [inline LevelMain.cpp]: Remove unneeded comment Check Sound Message

File: LevelMain.cpp [hunk 2]
```diff
@@ -134,15 +134,17 @@ Export void AIProc()
...
 {
-	LOCATION blightSpawnLoc(1 + X_, 1 + Y_);
-	// Setup Blight
-	GameMap::SetVirusUL(blightSpawnLoc, 1);  // Spawn the Blight
-	TethysGame::SetMicrobeSpreadSpeed(60);  	// Set the Blight's spread speed
+	LOCATION blightSpawnLoc(1 + X_, 255 + Y_);
+	
+
+	GameMap::SetVirusUL(blightSpawnLoc, 1); // Spawn the blight
+	TethysGame::SetMicrobeSpreadSpeed(60);	// set blight speed
```
Comment (Brett208) [inline LevelMain.cpp]: Unneeded comment set blight speed

**Review by Brett208 — COMMENTED** [review]
Couple of recommended changes to cleanup comments

File: LevelMain.cpp (hunk 1 above)
Comment (WoodardEric) [reply LevelMain.cpp]: done

**Review by Brett208 — APPROVED** [review]

**Pull request merged**